CMakeLists.txt

```diff
@@ -81,6 +79,19 @@ IF (ENABLE_DRPM)
     ADD_DEFINITIONS("-DCR_DELTA_RPM_SUPPORT")
 ENDIF (ENABLE_DRPM)
 
+# Zlib-ng is a performance-oriented fork of zlib
+# with a more active development cadence
+OPTION (WITH_ZLIBNG "Build against zlib-ng rather than zlib" OFF)
+IF (WITH_ZLIBNG)
+    pkg_check_modules(ZLIB-NG REQUIRED)
```

Comment:
Not sure if it's related but while this seems to work fine, none of the variables (like `ZLIBNG_INCLUDE_DIRS`, `ZLIBNG_LIBRARY`) get populated.